codalab/server/json_api_plugin.py

```diff
@@ -0,0 +1,110 @@
+"""
```

Comment:
Evaluate whether you think it's worth it. Are you planning on migrating the rest of the code to use the JSON API? If so, you're welcome to do it. If not, then here it's just overkill that makes the code hard to read unless you're pretty familiar with the JSON API.

Take for example the fieldset stuff. Is it really necessary? What's wrong with just sending all the data. I can see it being useful with some worksheet stuff where reading the extra info requires an extra database call. Even then, at this point it's probably premature optimization.